From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Patrick McCarty <patrick.mccarty@intel.com>
Date: Fri, 20 Sep 2019 15:39:12 -0700
Subject: [PATCH] Set chroot group to mockbuild

In order for stateless integration to work properly, we want to specify
an undefined group name ("mockbuild") instead of a defined group name
("mock").

Signed-off-by: Patrick McCarty <patrick.mccarty@intel.com>
---
 mock/py/mockbuild/util.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mock/py/mockbuild/util.py b/mock/py/mockbuild/util.py
index 8055780..c462dbd 100644
--- a/mock/py/mockbuild/util.py
+++ b/mock/py/mockbuild/util.py
@@ -943,7 +943,7 @@ def setup_default_config_opts(unprivUid, version, pkgpythondir, unprivGid=None):
         except KeyError:
             #  'mock' group doesn't exist, must set in config file
             pass
-    config_opts['chrootgroup'] = 'mock'
+    config_opts['chrootgroup'] = 'mockbuild'
     config_opts['chrootuser'] = 'mockbuild'
     config_opts['build_log_fmt_name'] = "unadorned"
     config_opts['root_log_fmt_name'] = "detailed"
